style.css

```diff
@@ -129,29 +145,63 @@ a {
 }
 
 .divChild2:hover {

```

Comment:
I love how you're making the element so dynamic! Changing the border radius is cool but a little unconventional. More common is to subtly change the colour of the button/font (although this works better maybe when the button is a darker colour), or add in a subtle border, etc.